wireapp/wire-ios-mono
# fix: Scrolling issues in team migration - WPB-15048

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-15048" title="WPB-15048" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-15048</a>  [iOS] Individual to team migration - Small UI Issues
  </td></table>
  <br />
 





### Issue

This PR adds a couple of fixes to scrolling in the team migration flow:

- Removes the scrolling indicators
- On larger devices ensures the "Continue" button is within the safe area - currently it is a little bit outside
- On smaller devices ensures that she scrolling goes to the bottom of the screen - currently it is clipped 24 points above the screen. See videos

| Old | New |
| --- | --- |
| <video src="https://github.com/user-attachments/assets/10c8df1f-cbad-423f-8ea5-06f84182a718"> |  <video src="https://github.com/user-attachments/assets/d0920335-5271-4d68-b6d2-40a8ed74b075"> |
| <video src="https://github.com/user-attachments/assets/4474593e-9b42-4967-a410-e322819a4be0"> |  <video src="https://github.com/user-attachments/assets/af0fe5e6-f6a9-4f42-91ba-0241f2037202"> |

### Testing

Repeat on small and large devices include iPhone SE

0. Ensure API v7 is in use and you are logged in with an individual account
1. Navigate to the profile screen
2. Tap "Create Wire Team"
3. Go throw the flow paying special attention how things scroll

---

### Checklist
